components/04-views/registration/_registration.scss

```diff
@@ -0,0 +1,215 @@
+        width: $registration__legend-width;
+        font-size: $registration__legend-font-size;
+        font-weight: $registration__legend-font-weight;
+        &:after,
+        &:before {
+            content: ' ';
+            position: absolute;
+        }
+        &:before {
+            width: $registration__legend-long-border-width;
+            height: $registration__legend-long-border-height;
+            bottom: $registration__legend-long-border-bottom-offset;
+            left: $registration__legend-long-border-left-offset;
+            background-color: $registration__legend-long-border-color;
+            z-index: 2;
+        }
+        &:after {
+            width: $registration__legend-short-border-width;
+            height: $registration__legend-short-border-height;
+            bottom: $registration__legend-short-border-bottom-offset;
+            left: $registration__legend-short-border-left-offset;
+            background-color: $registration__legend-short-border-color;
+            z-index: 3;
+        }
+    }
+    &__fieldset {
+        margin: $registration__fieldset-margin;
+        @include mq($screen-m) {
+            min-width: $registration__fieldset-min-width\@medium;
+            width: $registration__fieldset-width\@medium;
+        }
+    }
+    &__newsletter-checkbox {
+        margin: $registration__newsletter-checkbox-margin;
+    }
+    &__newsletter-checkbox-label {
+        padding-left: $registration__newsletter-checkbox-label-padding-left;
+        line-height: $registration__newsletter-checkbox-label-line-height;
+        &:before {
+            top: $registration__newsletter-checkbox-icon-offset;
+            left: $registration__newsletter-checkbox-icon-offset;
+        }
+        &:after {
+            width: $registration__newsletter-checkbox-icon-clickable-area;
+            height: $registration__newsletter-checkbox-icon-clickable-area;
+            content: ' ';
+            position: absolute;
+            top: 0;
+            left: 0;
+        }
+    }
+    &__newsletter-checkbox-field {
+        &:checked ~ .checkbox__label {
+            font-weight: $registration__newsletter-checkbox-checked-label-font-weight;
+        }
+    }
+    &__newsletter-checkbox-icon {
+        top: $registration__newsletter-checkbox-icon-offset;
+        left: $registration__newsletter-checkbox-icon-offset;
+    }
+    &__password {
+        margin: $registration__password-margin;
+    }
+    &__password-strength {
+        margin: $registration__password-strength-margin;
+    }
+    &__password-strength-progress {
+        border-radius: $registration__password-strength-radius;
+        background-color: $registration__password-bg-color;
+        padding: $registration__password-strength-padding;
+        color: $registration__password-color;
+        &:before {
+            .registration & {
+                background-color: $registration__password-bg-color;
+            }
+        }
+        span {
```

Comment:
😱  mother of specificities, do we really have to use this selector here ?

Reply:
sure not. thx. done.
ps. 😱  mother of professionalism, do we really add scary icons to code review comment?